### ISSUE TITLE
Persistant notification for reminders - (Feature request)

### ISSUE THREAD

**Issue by W1zardK1ng** [maltaisn/another-notes-app#129]
Currently the reminder notfications can be simply swiped away cauing to miss the action at times, Please add an option to enable persitant notifications so that it stay on the notification bar until its marked as done or postponed.

**Comment by maltaisn** [comment]
I don't think this should be a concern of the app. There's a notification history system option you can enable. Also, I think on-going (persistent) notifications can be dismissed as usual starting from Android 14 so it wouldn't solve the issue.
